webpack/webpack.config.prod.js

```diff
@@ -7,19 +7,15 @@ config.optimization = {
   ...config.optimization,
   flagIncludedChunks: false,
   concatenateModules: false,
-  minimize: process.env.NODE_ENV !== 'test',
+  // Chrome web store doesn't allow minified code
+  minimize: false,
   moduleIds: 'deterministic',
   splitChunks: {
     hidePathInfo: false,
     minSize: 10000,
     maxAsyncRequests: Infinity,
     maxInitialRequests: Infinity,
   },
-  minimizer: [
-    new ESBuildMinifyPlugin({
```

Comment:
Is there an import that needs to be removed because this has been removed? https://github.com/blockstack/stacks-wallet-web/blob/2ce91168dd71961723b46b90222fc2a91dec3d07/webpack/webpack.config.prod.js#L3